flexmeasures/api/v3_0/users.py

```diff
@@ -146,36 +153,28 @@ def index(
                     "last_login_at": naturalized_datetime_str(user.last_login_at),
                     "last_seen_at": naturalized_datetime_str(user.last_seen_at),
                 }
-                for user in paginated_users
+                for user in paginated_users.items
             ]
-            response: dict = {
+            response: dict | list = {
                 "users": users_response,
                 "total_items": total_items,
```

Comment:
You return `total_items`and `total_pages`  in your API, but does the DataTable use that?

In the Asset API, these terms are used:

```
response = {
                "data": asset_schema.dump(select_pagination.items, many=True),
                "num-records": num_records,
                "filtered-records": select_pagination.total,
            }
```

As the asset API docstring says, the response schema for pagination is inspired by https://datatables.net/manual/server-side#Returned-data

The assets.html code uses this mapping to get the DataTable to interpret the response correctly:

`callback({"data": clean_response, "recordsTotal": response["num-records"], "recordsFiltered": response["filtered-records"]});`

We should do something similar and also use the same terms in the API as we use over there (for consistency).